newspack-theme/inc/template-functions.php

```diff
@@ -630,6 +630,60 @@ function newspack_the_custom_logo() {
 	}
 }
 
+/**
+ * Customize the site title.
+ * This is the one that is displayed in the header, next to the logo, when "Display Site Title" is enabled in Appearance > Customize > Site Identity.
+ * If not enabled, this site title is still rendered, but hidden via CSS.
+ *
+ * @return string The site title.
+ */
+function newspack_custom_site_title() {
+	/**
+	 * Filters the site name.
+	 *
+	 * @param string $blog_name The site name.
+	 */
+	$blog_name = apply_filters( 'custom_site_title_name', get_bloginfo( 'name' ) );
+
+	if ( empty( $blog_name ) ) {
+		return;
+	}
+
+	/**
+	 * Filters the site URL.
+	 *
+	 * @param string $site_url The site URL.
+	 */
+	$site_url = apply_filters( 'custom_site_tile_url', home_url( '/' ) );
```

Comment:
```suggestion
	$site_url = apply_filters( 'newspack_custom_site_title_url', home_url( '/' ) );
```

Same as the other, plus a small typo